### PR TITLE
Fix errors at boot time

### PR DESCRIPTION
**Pull request by sabau**
_Description:_
After running a yarn build:gaia from develop I encountred some errors
- --moniker seems not to be a named parameter anymore
- tls seems to be disabled by default instead of what the --help states

❤️ Thank you!

---

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: tasks/gaia.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ async function initGenesis(
   nodeHomeDir
 ) {
   await makeExec(
-    `${nodeBinary} add-genesis-account ${address} 150stake,1000photino  --home ${nodeHomeDir}`
+    `${nodeBinary} add-genesis-account ${address} 200000000stake,1000photino  --home ${nodeHomeDir}`
```
Comment (jbibla) [inline tasks/gaia.js]: 🤑

**Comment by jbibla** [comment]
from our develop or from SDK develop? can we confirm these are intended changes, not just bugs or mistakes?

**Comment by jbibla** [comment]
also, changelog 😄

File: tasks/gaia.js (hunk 1 above)
Comment (sabau) [reply tasks/gaia.js]: richness is gooood, but this change happened because of an error in the creation under 100000000stake

**Comment by sabau** [comment]
From our develop, I removed the local-testnet folder and tried to build and start, then some errors occurred

**Comment by sabau** [comment]
The seed phrase was stripped recently as was printed in non standard way, I just created an issue to restore it https://github.com/cosmos/cosmos-sdk/issues/3518

**Comment by sabau** [comment]
> from our develop or from SDK develop? can we confirm these are intended changes, not just bugs or mistakes?

Yep, except the huge amount of stakes needed to boot. I shuould report also that one.

**Pull request merged**